frontend/web/components/EditPermissions.tsx

```diff
@@ -1176,29 +1190,63 @@ const EditPermissions: FC<EditPermissionsType> = (props) => {
                                     {email}
                                   </div>
                                 </Flex>
-                                {role === 'ADMIN' ? (
-                                  <Flex className='table-column fs-small lh-sm'>
-                                    <Tooltip
-                                      title={'Organisation Administrator'}
-                                    >
-                                      {
-                                        'Organisation administrators have all permissions enabled.<br/>To change the role of this user, visit Organisation Settings.'
-                                      }
-                                    </Tooltip>
-                                  </Flex>
-                                ) : (
-                                  <Flex
-                                    onClick={onClick}
-                                    className='table-column fs-small lh-sm'
-                                  >
-                                    {matchingPermissions &&
-                                    matchingPermissions.admin
-                                      ? `${Format.camelCase(
-                                          level,
-                                        )} Administrator`
-                                      : 'Regular User'}
-                                  </Flex>
-                                )}
+                                <div className='d-flex  gap-2 table-column fs-small'>
+                                  {role === 'ADMIN' ? (
+                                    <div className=''>
+                                      <Tooltip
+                                        title={
+                                          <PermissionSummaryItem isAdmin />
+                                        }
+                                      >
+                                        {
+                                          'Organisation administrators have all permissions enabled.<br/>To change the role of this user, visit Organisation Settings.'
```

Comment:
This tool tip looks a little odd with the single line break. We should either remove it, or add another imo. 

```suggestion
                                          'Organisation administrators have all permissions enabled. To change the role of this user, visit Organisation Settings.'
```